scripts/cmapi-start

```diff
@@ -8,6 +8,7 @@ API_KEY=${API_KEY:-somekey123}
 
 # Intialize Container If Necessary
 if [ ! -e $IFLAG ]; then
+  cp /opt/cmapi/engine_files/mcs-loadbrm.py /usr/bin/mcs-loadbrm.py
```

Comment:
Until new engine packages are ready, copy over the new mcs-loadbrm.py file shipped with cmapi.